packages/vc-handler-ld-local/src/ld-credential-module.ts

```diff
@@ -111,15 +133,28 @@ export class LdCredentialModule {
   ): Promise<boolean> {
     const verificationSuites = this.getAllVerificationSuites()
     this.ldSuiteLoader.getAllSignatureSuites().forEach((suite) => suite.preVerificationCredModification(credential))
-    const result = await vc.verifyCredential({
-      credential,
-      suite: verificationSuites,
-      documentLoader: this.ldDocumentLoader.getLoader(context, fetchRemoteContexts),
-      purpose: purpose,
-      compactProof: false,
-      checkStatus: checkStatus,
-    })
-
+    let result
+    if (credential.proof.type === 'BbsBlsSignature2020') {
+      //Should never be null or undefined
+      const suite = this.ldSuiteLoader
+        .getAllSignatureSuites()
+        .find((s) => s.getSupportedVeramoKeyType() === 'Bls12381G2')
+        ?.getSuiteForVerification() as BbsBlsSignature2020
+      result = await jsonld.verify(credential, {
```

Comment:
This misses stuff like revocationStatus and is not the same as vc.verifyCredential
https://github.com/digitalcredentials/vc-js/blob/9e40e67a3060289c899890c2cfb76681cb893368/lib/vc.js#L256

Reply:
Cannot use it because vc-js uses different packages

require('@digitalcredentials/jsonld');
const jsigs = require('@digitalcredentials/jsonld-signatures');

Reply:
Yes these those are forks of the same packages that can run on RN as well.

Obviosuly we cannot end up with missing pieces and different logic